trojsten/settings/production.py

```diff
@@ -1,28 +1,19 @@
+import warnings
+
 from django.core.exceptions import ImproperlyConfigured
 
 from trojsten.settings.common import *
 
 
 def requiredenv(name):
     if name not in os.environ:
-        raise ImproperlyConfigured("Value %s missing in environment configuration" % name)
+        warnings.warn("Value %s missing in environment configuration" % name)
     return os.environ.get(name)
 
 
-DEBUG = False
```

Comment:
is this intentional?

Reply:
Yes, this is default already. And I wanted to be able to override it using env var if needed.